test/e2e/storage/persistent_volumes-local.go

```diff
@@ -149,6 +150,7 @@ var (
 
 var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 	f := framework.NewDefaultFramework("persistent-local-volumes-test")
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
```

Comment:
oh, looks like this is because we're using hostexec to inspect stuff on the node...